File: Express-Node/src/routes/shoes.routes.js
import { Router } from "express";
import { createShoe, deleteShoe, getShoe, getShoes, updateShoe } from "../controllers/shoes.controller.js";

const router = Router();

router.get('/shoes', getShoes)
router.post('/shoes', createShoe)
router.get('/shoes/:id', getShoe)
router.put('/shoes/:id', updateShoe)
router.delete('/shoes/:id', deleteShoe)

export default router;

File: Python-Flask/src/routes/ShoesRoutes.py
from flask import Blueprint

from controllers.ShoesController import get_shoes_controller, create_shoe_controller, get_shoe_controller, delete_shoe_controller, update_shoe_controller

shoes = Blueprint('shoes', __name__)

@shoes.route('/', methods=['GET'])
def get_shoes():
  return get_shoes_controller()

@shoes.route('/<id>', methods=['GET'])
def get_shoe(id):
  return get_shoe_controller(id)

@shoes.route('/', methods=['POST'])
def create_shoe():
  return create_shoe_controller()

@shoes.route('/<id>', methods=['PUT'])
def update_shoe(id):
  return update_shoe_controller(id)

@shoes.route('/<id>', methods=['DELETE'])
def delete_shoe(id):
  return delete_shoe_controller(id)